bigchaindb/web/server.py

```diff
@@ -102,7 +102,10 @@ def create_server(settings):
         settings['workers'] = (multiprocessing.cpu_count() * 2) + 1
 
     if not settings.get('threads'):
```

Comment:
If we do remove the config option for threads this check is not needed since `threads` will never be set